automated-comments/csharp/general/use_string_interpolation_not_string_format.md

```diff
@@ -0,0 +1 @@
+You are currently using `string.Format` to dynamically build a string. This is perfectly fine, but the more idiomatic C# solution to create such a string is to use [string interpolation](https://csharp.net-tutorials.com/operators/the-string-interpolation-operator/). Note that string interpolation is just a compiler trick, as under the hood any string interpolation expression is converted to a `string.Format` call. The benefit of using string interpolation is thus purely visual.
```

Comment:
@mikedamay said:

“Note that string interpolation is just a compiler trick”. Maybe “Note that string interpolation is just a compiler trick, also known as syntactic sugar”?

Reply:
I've added the syntactic sugar hint, and did some rephrasing about the compiler bit.

Reply:
Sounds good